packages/orchestration/src/chain-info.js

```diff
@@ -64,6 +65,44 @@ const knownChains = /** @satisfies {Record<string, ChainInfo>} */ (
 
 /** @typedef {typeof knownChains} KnownChains */
 
+/**
+ * @param {ERef<import('@agoric/vats').NameHubKit['nameAdmin']>} agoricNamesAdmin
+ * @param {string} name
+ * @param {CosmosChainInfo} chainInfo
+ * @param {(...messages: string[]) => void} log
+ */
+export const registerChain = async (
```

Comment:
It's a little awkward that importing `registerChain` also grabs the big blob of well-known-chain info. The only thing that needs the big blob is a core-eval, while callers of `registerChain` are likely to be contracts.

Reply:
ah. I misread. thanks